pkg/driver/helmdriver.go

```diff
@@ -26,44 +26,49 @@ const (
 
 // helmDriver implements PackageDriver to install packages from Helm charts.
 type helmDriver struct {
-	cfg        *action.Configuration
-	secretAuth auth.Authenticator
-	log        logr.Logger
-	settings   *cli.EnvSettings
+	cfg              *action.Configuration
+	secretAuth       auth.Authenticator
+	kubeconfigClient auth.KubeconfigClient
+	log              logr.Logger
+	settings         *cli.EnvSettings
 }
 
 var _ PackageDriver = (*helmDriver)(nil)
 
-func NewHelm(log logr.Logger, secretAuth auth.Authenticator) (*helmDriver, error) {
-	settings := cli.New()
+func NewHelm(log logr.Logger, secretAuth auth.Authenticator, kubeconfigClient auth.KubeconfigClient) (*helmDriver, error) {
+	return &helmDriver{
+		secretAuth:       secretAuth,
+		kubeconfigClient: kubeconfigClient,
```

Comment:
Is the `kubeconfigClient` a required field for the NewHelm function now?

Reply:
yes

Reply:
Ok I'll need some changes in generatebundle, since we use it there too.